internal/database/access_tokens.go

```diff
@@ -202,19 +205,40 @@ func (s *accessTokenStore) createToken(ctx context.Context, subjectUserID int32,
 		// not been deleted. If they were deleted, the query will return an error.
 		`
 WITH subject_user AS (
-  SELECT id FROM users WHERE id=$1 AND deleted_at IS NULL FOR UPDATE
+  SELECT
+    id,
+	(
+		SELECT COUNT(*)
+		FROM access_tokens
+		WHERE
+		  subject_user_id = $1
+		  AND deleted_at IS NULL
+		  AND (expires_at IS NULL OR expires_at > NOW())
+		  AND internal IS NOT True
```

Comment:
pythonic nit:

```suggestion
		  AND internal IS NOT true
```

or

```suggestion
		  AND internal IS NOT TRUE
```